### PR TITLE
C_CPP : typedef

### PR DESCRIPTION
**Pull request by utkarshpathak2504**
<hr>

## Description 📜

Documentation for typedef  in C++ #3447  
contributed by:- Utkarsh Pathak
Batch:- C_CPP 5
@Saviour1001 Please check the typedef.md and let me know the changes if any.

<hr>

## Type of change 📝

- [X] Doc#umentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [X] C/#CPP

<hr>

## Checklist ✅

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [X] I have performed a self-review of my own code or work.
- [X] I have commented on my code, particularly in hard-to-understand areas.
- [X] My changes generates no new warnings.
- [X] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: C_CPP/Structure padding, Union, Enum, & typedef/typedef.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	integer num1, num2, sum;
+	
+	cout<<"Enter two number: ";
+	cin>>num1>>num2;
+	sum=num1+num2;
+	cout<<sum;
+
+	return 0;
+}
+```
```
Comment (vasu-1) [inline C_CPP/Structure padding, Union, Enum, & typedef/typedef.md:51]: Put some explanation in the code and remove unwanted spaces !!!

**Review by vasu-1 — CHANGES_REQUESTED** [review]
Do Some More Markdown Formatting !!!!

File: C_CPP/Structure padding, Union, Enum, & typedef/typedef.md (hunk 1 above)
Comment (utkarshpathak2504) [reply C_CPP/Structure padding, Union, Enum, & typedef/typedef.md:51]: > Do Some More Markdown Formatting !!!!

Thanks for reviewing, I will do the suggested changes.

**Comment by utkarshpathak2504** [comment]
Hi @vasu-1, I have made the required changes, please have a look and let me know.

**Review by vasu-1 — APPROVED** [review]
Good work.
@Suvraneel good to go for merge

**Comment by utkarshpathak2504** [comment]
> Good work. @Suvraneel good to go for merge

Thank you!

**Pull request merged**